platform/barefoot/bfn-sai.mk

```diff
@@ -1,5 +1,11 @@
 BFN_SAI = bfnsdk_1.0.0_amd64.deb
-$(BFN_SAI)_URL = "https://github.com/barefootnetworks/sonic-release-pkgs/raw/sde-sai1.3.3/bfnsdk_1.0.0_amd64.deb"
 
+ifeq ($(CONFIGURED_P4_STACK),p4_16)
```

Comment:
Since we do not plan to merge to the upstream p4_16/p4_14 stack selection infrastructure at this moment, probably we can use the same approach but without mentioning p4_16, just to expose the approach itself. Also, we can use this methodology to either build from public DEB package or from the sources $(BFN_SAI)_PATH. Again, without mentioning p4_16.

Reply:
Yes it will have to be transparent. For same switch.p4 profile, we dont need to maintain both p4-14 and p4-16 versions.